plugins/es-repository-azure/src/test/java/org/elasticsearch/repositories/azure/AzureSnapshotIntegrationTest.java

```diff
@@ -150,6 +150,27 @@ public void create_azure_snapshot_and_restore_with_secondary_endpoint() {
         handler.blobs().keySet().forEach(x -> assertThat(x).doesNotEndWith("dat"));
     }
 
+    @Test
+    public void create_azure_repo_with_sas_token() {
+        execute("CREATE REPOSITORY r1 TYPE AZURE WITH (" +
+            "container = '" + CONTAINER_NAME + "', " +
+            "account = 'devstoreaccount1', " +
+            "sas_token = 'aaQc3RvcmVhY2NvdW50Mdhdhd', " +
+            "location_mode = 'PRIMARY_ONLY', " +
+            "endpoint = '" + httpServerUrl() + "')");
+        assertThat(response.rowCount()).isEqualTo(1L);
+    }
+
+    @Test
+    public void testCreateAzureRepoWithMissingMandatorySettings() {
```

Comment:
very minor, but I think we once agreed to use lowercase with underscores names.
```suggestion
    public void test_create_azure_repo_with_missing_mandatory_settings() {
```